aws/data_source_aws_instance_test.go

```diff
@@ -502,3 +580,25 @@ data "aws_instance" "foo" {
   instance_id = "${aws_instance.foo_instance.id}"
 }
 `
+
+func testAccInstanceDataSourceConfig_getPasswordData(val bool) string {
+	return fmt.Sprintf(`
+	resource "aws_key_pair" "foo" {
+		key_name = "tf-acc-winpasswordtest"
```

Comment:
The key name needs to be randomized for parallel or broken test runs. `key_name = "tf-acc-%d"` (where `%d` can come from the top level test function, e.g. `rInt := acctest.RandInt()`